src/scripts/main.js

```diff
@@ -1,3 +1,69 @@
 'use strict';
 
-// write code here
+const appendRowBtn = document.querySelector('.append-row');
+const removeRowBtn = document.querySelector('.remove-row');
+const appendColBtn = document.querySelector('.append-column');
+const removeColBtn = document.querySelector('.remove-column');
+const table = document.querySelector('.field');
+const minRows = 2;
+const maxRows = 10;
+const minCols = 2;
+const maxCols = 10;
+let rowCount = table.rows.length;
+let colCount = table.rows[0].cells.length;
```

Comment:
Ensure that the table has at least one row and one column initially. If the table is empty, accessing `table.rows[0].cells.length` will throw an error. Consider checking if the table has rows and columns before accessing them.